src/util/helpers.tsx

```diff
@@ -290,3 +290,27 @@ export const getClientOS = (userAgent: string) => {
 
   return null;
 };
+
+export const getImageToTarballBody = (fileList: FileList): File | FormData => {
+  let obj: File | FormData | null = null;
+  const formData = new FormData();
+
+  if (fileList.length === 1) {
+    obj = fileList[0];
```

Comment:
```suggestion
    return fileList[0];
```

This way we don't need the else, and we don't need to define obj above. We can directly return formData below.